##HerkuleX_Info Array_EEP

int8 index
MsgHerkuleX_EEP[] pHerkuleX_EEP

